src/build.cc

```diff
@@ -168,6 +168,9 @@ string BuildStatus::FormatProgressStatus(
   string out;
   char buf[32];
   int percent;
+  char total_edges_str[32];
+  snprintf(total_edges_str, sizeof(total_edges_str), "%d", total_edges_);
```

Comment:
I think that this is not a problem. It is just used as an estimate for length alignment. It just provide better output for next print...